yt_dlp/extractor/common.py

```diff
@@ -2536,26 +2545,79 @@ def _extract_mpd_formats_and_subtitles(
             errnote='Failed to download MPD manifest' if errnote is None else errnote,
             fatal=fatal, data=data, headers=headers, query=query)
         if res is False:
-            return [], {}
+            return []
         mpd_doc, urlh = res
         if mpd_doc is None:
-            return [], {}
+            return []
 
         # We could have been redirected to a new url when we retrieved our mpd file.
         mpd_url = urlh.url
         mpd_base_url = base_url(mpd_url)
 
-        return self._parse_mpd_formats_and_subtitles(
-            mpd_doc, mpd_id, mpd_base_url, mpd_url)
+        return self._parse_mpd_periods(mpd_doc, mpd_id, mpd_base_url, mpd_url)
 
     def _parse_mpd_formats(self, *args, **kwargs):
         fmts, subs = self._parse_mpd_formats_and_subtitles(*args, **kwargs)
         if subs:
             self._report_ignoring_subs('DASH')
+        if any(f.get('is_dash_periods') for f in fmts):
+            self.report_warning(bug_reports_message(
+                'Ignoring multiple periods found in the DASH manifest; '
+                'if part of the video is missing,'
+            ), only_once=True)
```

Comment:
Pls explain

Reply:
Yes, a leftover from an earlier version. I have removed it.